### DISCUSSION TITLE
Unwanted HTML element created when closing modal

### DISCUSSION THREAD

**Discussion by GuilhermeZaninPs** [mdbootstrap/TW-Elements#1437] [Support from community]
Every time I close a modal in my web application, an HTML element <fictum style=""></fictum> is created. I'm not sure why this is happening, and I would like to understand the root cause of the issue and find a way to prevent it from happening.

Steps to reproduce:

Open the modal by clicking on a button.
Close the modal by clicking on the close button.
Check the HTML code of the page and notice that a <fictum style=""></fictum> element has been created.
Expected behavior:

When I close the modal, no unwanted HTML elements should be created.

Actual behavior:

An HTML element <fictum style=""></fictum> is created every time I close the modal.

Additional information:

I'm using React js.

Please let me know if there's any other information I can provide to help diagnose and solve this issue.

**Comment by Trochonovitz** [comment]
Hi, what HTML element are you talking about? Could you provide some more informations?
